File: Ch1-Fundamentals/0006-You-Say-its-Your-Birthday/main.py
birth_month = 10
birth_day = 29

def check_birth_date( month, day ):
    if (( month < 1 or month > 12 ) or ( day < 1 or day >31 )):
        print("Values out of range!")
        return False
    
    if (( month == birth_month and day == birth_day ) or ( month == birth_day and day == birth_month )):
        print(f"How did you know? It's {month}/{day}!")
        return True
    print("Just another day...")
    return False

check_birth_date(10, 27)
check_birth_date(17, 4)
check_birth_date(10, 29)
check_birth_date(13, 3)
check_birth_date(4, 17)

File: Ch1-Fundamentals/0006-You-Say-its-Your-Birthday/main.js
let birthMonth = 10
let birthDay = 29

function checkBirthDate( month, day ) {
    if (( month < 1 || month > 12 ) || ( day < 1 || day > 31 )) {
        console.log("Values out of range!")
        return false;
    }
    
    if (( month === birthMonth && day === birthDay ) || ( month === birthDay && day === birthMonth )) {
        console.log(`How did you know? It's ${month}/${day}!`);
        return true;
    }
    console.log("Just another day...");
    return false;
}

checkBirthDate(10, 29);
checkBirthDate(30, 1);
checkBirthDate(1, 32);
checkBirthDate(4, 17);
checkBirthDate(10, 27);